proxmox/resource_vm_qemu.go

```diff
@@ -1873,21 +1869,6 @@ func resourceVmQemuRead(ctx context.Context, d *schema.ResourceData, meta interf
 	// Reset reboot_required variable. It should change only during updates.
 	d.Set("reboot_required", false)
 
-	// Pool
```

Comment:
Kinda confused why this was even here line `1867 1863` already sets this property.